### DISCUSSION TITLE
does Filament version 3 Tabs support multi table insertion

### DISCUSSION THREAD

**Discussion by Hegabovic** [filamentphp/filament#10907] [Help]
i have a service resource, in its form build , having two tabs 
 - first tab : to create form  like name , age , ...etc , this will be saved in services table
 - second tab: to save configuration in another database table for example called : configurations table 
 
 so does filament support that ??

**Comment by leandrocfe** [comment]
Customize the creation process
https://filamentphp.com/docs/3.x/panels/resources/creating-records#customizing-the-creation-process
